tests/test_corpus/en_ewt-ud-dev.txt

```diff
@@ -0,0 +1,52 @@
+From the AP comes this story : President Bush on Tuesday nominated two
```

Comment:
Just checked:
```Attribution — You must give appropriate credit, provide a link to the license, and indicate if changes were made. You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use.```

Please add a README.md in this folder, specifying the source (I believe https://github.com/UniversalDependencies/UD_English-EWT/) and License. 

I know this is used just for testing, but it will be included in the NLP-Cube repo.

Reply:
Well, true. I'll add a license file. 

The en-ewt corpus is licensed under , meaning  (citation):

You are free to:

    Share — copy and redistribute the material in any medium or format
    Adapt — remix, transform, and build upon the material
    for any purpose, even commercially.
The licensor cannot revoke these freedoms as long as you follow the license terms:
    Attribution — You must give appropriate credit, provide a link to the license, and indicate if changes were made. You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use.
    ShareAlike — If you remix, transform, or build upon the material, you must distribute your contributions under the same license as the original. 

Sooo, I guess we're ok as long as we pass the license in the folder. I'll return with the file with a new commit shortly.